bundle/manifests/ingressnodefirewall.openshift.io_ingressnodefirewalls.yaml

```diff
@@ -253,11 +253,13 @@ spec:
                           items:
                             type: string
                           type: array
+                          x-kubernetes-list-type: atomic
```

Comment:
Does the change to `x-kubernetes-list-type: atomic` have anything to do with bpfman?

Reply:
no this is just recent tooling changes